packages/oas-to-har/package.json

```diff
@@ -50,17 +50,15 @@
   },
   "dependencies": {
     "@readme/data-urls": "^3.0.0",
-    "lodash.get": "^4.4.2",
-    "lodash.set": "^4.3.2",
+    "lodash-es": "^4.17.21",
```

Comment:
didn't we have issues with this package here in the past where it **only** supports ESM?

Reply:
(sigh) I can't say I remember for sure... I think https://github.com/readmeio/oas-to-har/commit/a3421796c3a90fee3d8720c26c4f03df79cdaacb was actually the issue?

Reply:
worst-case scenario we use `tsup` for `oas-to-har` so we could probably bundle these directly if it ends up being a problem